tests/test_logging_config.py

```diff
@@ -134,6 +134,7 @@ def settings_context(content, directory=None, name='LOGGING_CONFIG'):
     sys.path.append(settings_root)
 
     with conf_vars({('logging', 'logging_config_class'): module}):
+        os.environ["AIRFLOW__LOGGING__LOGGING_CONFIG_CLASS"] = module
```

Comment:
We should reset this to it's old value after the test/block is over, and we probably don't need conf_vars too.

And we should have a comment here saying why, in this one case, we aren't using conf_vars.